.github/workflows/build-docker-image-and-binaries.yaml

```
@@ -15,6 +15,10 @@ on:
         description: Whether to enable wasm ("true"/"false")
         required: true
         default: "true"
+      ibc-hooks:
```

```suggestion
      ibc-wasm-hooks:
```
for consistency perhaps